core/vm/contracts.go

```diff
@@ -722,3 +754,81 @@ func (c *hashHeader) Run(input []byte, caller common.Address, evm *EVM, gas uint
 
 	return hashBytes, gas, nil
 }
+
+type getParentSealBitmap struct{}
+
+func (c *getParentSealBitmap) RequiredGas(input []byte) uint64 {
+	return params.GetParentSealBitmapGas
+}
+
+// Return the signer bitmap from the parent seal of a past block in the chain.
+// Requested parent seal must have occurred within 4 epochs of the current block number.
+func (c *getParentSealBitmap) Run(input []byte, caller common.Address, evm *EVM, gas uint64) ([]byte, uint64, error) {
+	gas, err := debitRequiredGas(c, input, gas)
+	if err != nil {
+		return nil, gas, err
+	}
+
+	// input is comprised of a single argument:
+	//   blockNumber: 32 byte integer representing the block number to access
+	if len(input) < 32 {
+		return nil, gas, ErrInputLength
+	}
+
+	blockNumber := new(big.Int).SetBytes(input[0:32])
+
+	// Ensure the request is for information from a previously sealed block.
+	if blockNumber.Cmp(evm.Context.BlockNumber) >= 0 {
```

Comment:
What does the parent seal bitmap look like for block 0?

Reply:
It's invalid to request from block 0. I've added an error.